src/App.js

```diff
@@ -1,25 +1,74 @@
-import logo from './logo.svg';
 import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Component } from 'react';
+import axios from 'axios';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Container from 'react-bootstrap/Container';
+import FloatingLabel from 'react-bootstrap/FloatingLabel';
+import Image from 'react-bootstrap/Image';
+import Navbar from 'react-bootstrap/Navbar';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      exploreQuery:'',
+      location:{},
+      image:'',
+      error:'',
+    };
+  }
+
+  handleButtonClick = async () => {
+    try {
+      const API_URL = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.exploreQuery}&format=json`;
+      const response = await axios.get(API_URL);
+      console.log(response.data);
+      this.setState({location:response.data[0]});
+      this.setState({image:`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_CITY_KEY}&center=${this.state.location.lat},${this.state.location.lon}&zoom=15`});
+    } catch {
+      this.setState({error: true});
+    }
+  }
+
```

Comment:
Nice work on incorporating try catch blocks! :)

When you can make sure to read a bit more on these and how they cause some issues in React. Really nice working this in 👍 

Note: Just remove those helpful logs when you have things a workin'/before you merge PR :)